install/grist-install.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/gristlabs/grist-core
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  sudo \
+  make \
+  gnupg \
+  ca-certificates \
+  mc \
```

Comment:
I don't know if this is required. `homarr` (and other scripts) seem to add it as a dep, but I don't know how/if it's used. Maybe it's more of a courtesy include for the community?

Reply:
Yes exactly. tteck started it and we keep it.